repose-aggregator/src/docs/asciidoc/filters/index.adoc

```diff
@@ -95,6 +95,9 @@ Validates calls to an API by checking the URI format and HTTP method and enforci
 == <<slf4j-http-logging.adoc#,SFL4J HTTP Logging Filter>>
 *IN PROGRESS*
 
+== <<split-header.adoc#,Split Header Filter>>
+Splits a single line header with multiple values into multiple header lines each with a single value.
```

Comment:
They  don't necessarily have to be a single line. you could have two lines with five values

Reply:
True, although I think I underspecified here. This filter really will only split single line, multi-valued headers, but it will split _all_ single line, multi-valued headers (that are configured).

I'll think of some better words and update this.